src/shared/components/linkedIconCard/linkedIconCard.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './linkedIconCard.css';
+
+// Stateless component that wraps a link around an icon / img and title
+// Image or Icon are passed as a child to allow more flexibility
+// Props = url, title, and at least one child
+const linkedIconCard = props => (
+  <a href={props.url} className={styles.linkedIconCard}>
+    {props.children}
+    <span className={styles.linkedIconCard__title}>{props.title}</span>
+  </a>
+);
+
+linkedIconCard.propTypes = {
+  url: PropTypes.string.isRequired,
```

Comment:
I like the idea of this being used as a reusable component. What if we took it a step further? I already have something in play on the Contact page and the Landing page contains similar component (Font Awesome icon with text below it).

What if instead of this proptype structure, we did:
title, subtext (optional), fontAwesomeIcon (see how we can pass it explicitly instead of children), url (optional)

If you do it that way, we can make an issue to replace landing and contact page uses of this structure with the reusable component in place. As a random tip that could be tricky to see, if somebody passes a `subtext`, then the card's height should be unbound.

If you _really_ wanna get into it, you could utilize the tips [laid out here](https://github.com/OperationCode/operationcode_frontend/issues/264) and limit the characters that can be sent in the title and subtext.